tests/parm/field_table/field_table_hrrr

```diff
@@ -0,0 +1,65 @@
+# added by FRE: sphum must be present in atmos
```

Comment:
Now is not the time for last-minute changes. This is a minor issue which could be fixed in a later commit. This PR will be delayed again if you ask for another batch of last-minute changes.

Reply:
Also, this test must have its own field table so that the test case does not change at all if another test case changes. It is supposed to reproduce the inputs and outputs of the RRFS dev1 parallel exactly.

Reply:
@SamuelTrahanNOAA I think it will save time if we can fix it now rather than creating a new PR and running another round of tests since it is minor change. Thanks

Reply:
No, the tables are not identical.

```
  "TRACER", "atmos_mod", "sphum"
            "longname",     "specific humidity"
            "units",        "kg/kg"
-       "profile_type", "fixed", "surface_value=3.e-6" /
+       "profile_type", "fixed", "surface_value=1.e-6" /
```

Reply:
The new field table does not change the results, so I removed it.